src/core/Wizard/Step.tsx

```diff
@@ -64,12 +63,11 @@ export const Step = (props: StepProps) => {
   };
 
   const stepShape = (
-    <span
+    <li
       className={cx(
-        'iui-wizards-step',
+        'iui-wizard-step',
         {
-          'iui-wizards-step-completed': isPast,
-          'iui-wizards-step-current': isActive,
+          'iui-current': type !== 'workflow' && isActive,
           'iui-clickable': !!onClick && isPast,
```

Comment:
wofklow does not need it

Reply:
Moved `type !== 'workflow'` into `isPast` so it will be handled